com.unity.multiplayer.mlapi/Runtime/Messaging/InternalMessageHandler.cs

```diff
@@ -44,6 +44,25 @@ internal static void HandleConnectionRequest(ulong clientId, Stream stream)
 #if DEVELOPMENT_BUILD || UNITY_EDITOR
             s_HandleConnectionRequest.Begin();
 #endif
+            if (NetworkManager.Singleton.PendingClients.TryGetValue(clientId, out PendingClient client))
+            {
+                if (client.HasSentConnectionRequest)
+                {
+                    if (NetworkLog.CurrentLogLevel <= LogLevel.Normal)
```

Comment:
should this be `NetworkLog.CurrentLogLevel <= LogLevel.Normal` or just `< LogLevel.Normal`? the main reason why I ask this question is the fact that this fix is trying to mitigate DDOS / conn request abuse attacks and spamming lots of logs might not be the best idea still (IMHO).